app/src/main/assets/chart.html

```diff
@@ -2,11 +2,7 @@
 <link rel="stylesheet" href="chart.css">
```

Comment:
The chart.html is missing the following tags:
<!DOCTYPE html>
<html>
<meta charset="UTF-8">

This is good to assure the proper rendering of the page.